Vienna/Sources/Shared/StringExtensions.m

```diff
@@ -319,6 +320,10 @@ +(NSString *)stringByRemovingHTML:(NSString *)theString
 				isInTag = NO;
 				isInQuote = NO;	// Fix problem with Tribe.net feeds that have bogus quotes in HTML tags
 			}
+			else if (!isInQuote)
```

Comment:
Can you please move the `{` to the same line as the `else if`? :) Everything else looks good!

Reply:
Doing just that would lead to coexistence of inconsistent styles inside the block, which I find harder to read.

I therefore prefer using uncrustify to apply our recommended style to the whole method.